src/api/routes/index.js

```diff
@@ -0,0 +1,21 @@
+import { Router } from 'express';
+import photosController from 'controllers';
+import { BAD_REQUEST } from 'constants/error.types';
+
+const photosRouter = Router();
+
+photosRouter.get('/', async (req, res) => {
```

Comment:
Could you refactor this to follow the REST principles?

Reply:
I think what @marissacola is asking is what you would do to refactor the API to be more RESTful, with regard to your usage of resource nouns and parameters - or in other words how would you make your routes more REST-like and less RPC-like.

Reply:
Ok I understand the question.

I would refactor the routes to `photos/cats`, `photos/sharks` and just `/photos` (for a mix of both cats and sharks) to match the REST guidelines.